dockstore-webservice/src/main/java/io/dockstore/webservice/core/Workflow.java

```diff
@@ -249,6 +254,13 @@ public void setDefaultWorkflowPath(String defaultWorkflowPath) {
         getDefaultPaths().put(this.getDescriptorType().getFileType(), defaultWorkflowPath);
     }
 
+    @JsonProperty
+    public String getForumUrl() {
+        return forumUrl;
+    }
+    public void setForumUrl(String forumUrl) {
+        this.forumUrl = forumUrl; }
```

Comment:
Closing brace on its own line is the normal pattern.

Reply:
ugh, gotta fix checkstyle